playbook/app/pb_kits/playbook/pb_background/docs/_background_image.jsx

```diff
@@ -1,36 +1,101 @@
-import React from 'react'
+import React, { useState }  from 'react'
 import { Background } from '../..'
-import { Card, Flex, FlexItem, Title } from '../..'
-
-const BackgroundImage = (props) => (
-  <Background imageUrl="https://images.unsplash.com/photo-1528459801416-a9e53bbf4e17?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80">
-    <Flex
-        orientation="column"
-        vertical="center"
-        {...props}
-    >
-      <FlexItem>
-        <Title
-            dark
-            padding="lg"
-            size={1}
-            text="Background Kit Image"
-            {...props}
-        />
-      </FlexItem>
-      <FlexItem
-          padding="lg"
+import { Button, Card, Flex, FlexItem, Select, Title } from '../..'
+
+const BackgroundImage = (props) => {
+  const [transition, setTransition] = useState('')
+  const [apply, setApply] = useState({
+    imageUrl: '',
+    transition: '',
+  })
+
+  const loadImage = () => {
```

Comment:
@jon1735 We shouldn't have to do this much logic in the `docs/` example. 
This logic should be moved to the actual core component files for rails/react... the jsx and html.erb files. 

Can you sync over this tomorrow morning?

Reply:
To answer this after our pairing session this morning, we removed the button entirely from the doc example. This cleaned up the code a bit. Thanks again for your help :)